compose-jetsnack/README.md

```diff
@@ -1,80 +1,121 @@
+
+Compose Multiplatform for web is based on [Kotlin/Wasm](https://kotl.in/wasm), the newest target for Kotlin Multiplatform projects.
+It allows you to run your code in the browser with all WebAssembly's benefits, such as high and predictable application performance.
 
-> **Note**
-> Web support is Experimental and may be changed at any time. Use it only for evaluation purposes.
-> We would appreciate your feedback on it in the public Slack channel [#compose-web](https://slack-chats.kotlinlang.org/c/compose-web).
-> If you face any issues, please report them on [GitHub](https://github.com/JetBrains/compose-multiplatform/issues).
+In the next sections, you can find information to try out the Jetsnack application built with Compose Multiplatform for web and Kotlin/Wasm.
 
-Compose for Web is based on [Kotlin/Wasm](https://kotl.in/wasm), the newest target for Kotlin Multiplatform projects.
-It allows you to run your code in the browser with all the benefits that WebAssembly has to offer, such as good and predictable performance for your applications.
+## Set up the environment
 
-## Setup environment
+Before you start, prepare the IDE and browser you require to run the application.
 
 ### IDE
 
 We recommend using [IntelliJ IDEA 2023.1 or later](https://www.jetbrains.com/idea/) to work with the project.
-It has Kotlin/Wasm support out of the box.
+It supports Kotlin/Wasm out of the box.
 
 ### Browser (for Kotlin/Wasm target)
 
-To run applications built with Kotlin/Wasm in a browser, you need a browser supporting [wasm garbage collection feature](https://github.com/WebAssembly/gc):
+To run applications built with Kotlin/Wasm in a browser, you need a browser supporting the [Wasm Garbage Collection (GC) feature](https://github.com/WebAssembly/gc):
+
+**Chrome and Chromium-based**
+
+* **For version 119 or later:**
+
+  Works by default.
+
+**Firefox**
 
-- For **Chrome** and **Chromium-based** browsers (Edge, Brave etc.), it **should just work** since version 119.
-- For **Firefox** 120 it **should just work**.
-- For **Firefox** 119:
-    1. Open `about:config` in the browser.
-    2. Enable **javascript.options.wasm_gc**.
-    3. Refresh the page.
+* **For version 120 or later:**
 
+  Works by default.
 
-For more information see https://kotl.in/wasm_help/.
+**Safari/WebKit**
+
+Wasm GC support is currently under
+[active development](https://bugs.webkit.org/show_bug.cgi?id=247394).
+
+> **Note:**
+> For more information about the browser versions, see the [Troubleshooting documentation](https://kotl.in/wasm_help/).
 
 ## Build and run
 
-Check out the repository, navigate to the project folder, and use the following commands:
+To build and run the Jetsnack application with Compose Multiplatform for web and Kotlin/Wasm:
+
+1. In IntelliJ IDEA, open the repository.
+2. Navigate to the `compose-jetsnack` project folder.
+3. Run the application by using the commands explained in the following sections.
```

Comment:
same as above - consider combining the two sections into an unordered list and updating step 3